CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ Future Todo List
 - Update to use latest v2 PIE design tokens
 
 
+v10.0.0
```

Comment:
More of a question but most of the changes are reference changes (no impact to how this package is used in consuming apps) should we therefore us a minor bump? @ashleynolan.

Reply:
If consumers of fozzie don't need to change anything, then it should just be a minor version bump.

I suspect that as users rely on `pie-design-tokens` though and that has breaking changes, we'll likely need to bump the major version for those changes (if someone is referencing `$color-grey` for instance, they'll need to update stuff on their side.